commit 659e29f32e15d28408fdaef89150038fa9bf2976
Author: Umaro <umaro@narshe.net>
Date:   Tue Jun 21 21:12:45 2011 +0200

    vmalloc: remove redundant unlikely()
    
        IS_ERR() already implies unlikely(), so it can be omitted here.
    
        Signed-off-by: Tobias Klauser <tklauser@distanz.ch>
        Reviewed-by: Minchan Kim <minchan.kim@gmail.com>
        Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
        Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index 20a402c..f5f3522 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -739,7 +739,7 @@ static struct vmap_block *new_vmap_block(gfp_t gfp_mask)
 	va = alloc_vmap_area(VMAP_BLOCK_SIZE, VMAP_BLOCK_SIZE,
 					VMALLOC_START, VMALLOC_END,
 					node, gfp_mask);
-	if (unlikely(IS_ERR(va))) {
+	if (IS_ERR(va)) {
 		kfree(vb);
 		return ERR_PTR(PTR_ERR(va));
 	}
